newrelic/api/opentelemetry.py

```diff
@@ -238,19 +244,53 @@ def is_recording(self):
         if getattr(self.nr_trace, "end_time", None):
             return False
 
-        return getattr(self.nr_transaction, "priority", 1) > 0
+        # If priority is either not set at this point
```

Comment:
I think this is perhaps a merge conflict cause this is the same logic as the previous code except if it's None we set it to 1?

Reply:
I realized that this does something slightly different than that: the `transaction.priority` attribute is always going to exist, but it is initially set to `None`.  This `getattr` function only returns 1 if it does not have the `priority` attribute at all.

So, when it is set to `None` and it tries to do a comparison, it fails.